examples/jpeg2avifex.c

```diff
@@ -0,0 +1,99 @@
+/**
+ * A short program which converts a .jpg file into a .avif file -
+ * just to get a little practice with the basic functionality.
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif /* HAVE_CONFIG_H */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#ifdef HAVE_UNISTD_H
+#include <unistd.h>
+#endif
+
+#include "gd.h"
+
+#ifdef HAVE_LIBAVIF
+#include <avif/avif.h>
```

Comment:
do you need this include ?  doesn't seem like it.

Reply:
I guess I don't!

Reply:
argh